### PR TITLE
add optional message to addMembers and removeMembers channel methods

### PR DESCRIPTION
**Pull request by ravlio**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Comment by tbarbugli** [comment]
@ravlio looks good to me, why is it a draft?

File: src/channel.js [hunk 1]
```diff
@@ -241,9 +241,10 @@ export class Channel {
 		return data;
 	}
 
-	async addMembers(members) {
+	async addMembers(members, message) {
```
Comment (vishalnarkhede) [inline src/channel.js]: Would be nice to have short function doc -`members` param is obvious, but short explanation about message will be helpful I think!!

Something similar to:

Screen Shot 2019-11-05 at 11 12 23 AM

**Comment by vishalnarkhede** [comment]
LGTM !! Just have one small comment.

There is duplicate ticket on JIRA for this story - [CHAT-748](https://stream-io.atlassian.net/browse/CHAT-748?atlOrigin=eyJpIjoiMmY5OWZiZDk0YTVlNDBjMDhmNjEwYzdiZGNkMjNlZTgiLCJwIjoiaiJ9). Better to close/cancel it in this case with a comment :)

File: src/channel.js [hunk 2]
```diff
@@ -190,8 +190,8 @@ export class Channel {
 	/**
 	 * update - Edit the channel's custom properties
 	 *
-	 * @param {object} custom The object to update the custom properties of this channel with
-	 *
+	 * @param {object} channelData The object to update the custom properties of this channel with
+	 * @param {object} updateMessage Optional message object for channel members notification
```
Comment (vishalnarkhede) [inline src/channel.js]: Awesome :D

**Pull request merged**